contents/docs/advanced/proxy/nextjs.mdx

```diff
@@ -23,6 +23,10 @@ const nextConfig = {
         source: "/ingest/:path*",
         destination: "https://us.i.posthog.com/:path*",
       },
+      {
+        source: "/decide/:path*",
+        destination: "https://us.i.posthog.com/:path*",
```

Comment:
I'm no expert but I think it's
```suggestion
        destination: "https://us.i.posthog.com/decide/:path*",
```

Looks like you can visit `https://us.i.posthog.com/decide` in your browser

You can probably get away without the `:path*` in both cases because I'm not sure decide has any other paths

Reply:
yeah, you're right about the paths.  I'll remove it to avoid any confusion.